pilot/docker/Dockerfile.proxyv2

```diff
@@ -0,0 +1,30 @@
+FROM istionightly/base_debug
```

Comment:
Do we really want our production Dockerfiles extending from `base_debug`?

Reply:
That's what we use today - I just moved it to a separate file. 
The 'base_debug' is just xenial updated daily, plus basic debug tools.